src/Service.Tests/GraphQLBuilder/MultipleMutationBuilderTests.cs

```diff
@@ -349,9 +350,28 @@ public static async Task InitializeAsync()
         private static RuntimeConfigProvider GetRuntimeConfigProvider()
         {
             TestHelper.SetupDatabaseEnvironment(databaseEngine);
-            // Get the base config file from disk
             FileSystemRuntimeConfigLoader configPath = TestHelper.GetRuntimeConfigLoader();
-            return new(configPath);
+            RuntimeConfigProvider provider = new(configPath);
+
+            RuntimeConfig runtimeConfig = provider.GetConfig();
+
+            // Enabling multiple create operation because all the validations in this test file are specific
+            // to multiple create operation.
+            runtimeConfig = runtimeConfig with
+            {
+                Runtime = new RuntimeOptions(Rest: runtimeConfig.Runtime.Rest,
+                                                                GraphQL: new GraphQLRuntimeOptions(MultipleMutationOptions: new MultipleMutationOptions(new MultipleCreateOptions(enabled: true))),
+                                                                Host: runtimeConfig.Runtime.Host,
+                                                                BaseRoute: runtimeConfig.Runtime.BaseRoute,
+                                                                Telemetry: runtimeConfig.Runtime.Telemetry,
+                                                                Cache: runtimeConfig.Runtime.Cache)
+            };
+
+            System.IO.Abstractions.TestingHelpers.MockFileSystem fileSystem = new();
```

Comment:
It would be helpful to add a comment as to what the next few lines of code are doing.

Reply:
Added